### ISSUE TITLE
keyword allowing is not stable

### ISSUE THREAD

**Issue by dervisbey** [proginosko/LeechBlockNG-chrome#27]
I have a configuration that is specified such that every page should be blocked except some website and keywords (in advanced options, I clicked on "Treat keywords as allow-conditions rather than block-conditions"). So it looks like:

```
*
+wikipedia.org
+google.com
+...
.
.
.
~python
~messaging
~...
.
.
.

```
(I have a lot of exceptions, I don't know if it does play a role).

In general it works, but weirdly not for all keywords. E.g. python keyword works, I can open:
https://www.activestate.com/products/python/
But the messaging keyword e.g. doesn't work, I cant open:
https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingComponentsIntro.html

Why is that?

Edit:
well, rn I cant open the python website anymore. The keywords are generally unstable 
Edit2:
maybe another example. I have allowed ~EPEX and opening the following website works:
https://acronyms.thefreedictionary.com/EPEX

**Comment by proginosko** [comment]
Thanks for the report. I was able to reproduce this issue. I think I know why it is happening and I will work on a fix.

**Comment by proginosko** [comment]
This should be fixed in version 1.0.11, currently awaiting review. If the problem persists, please reopen this issue.
